README.md

```diff
@@ -100,35 +102,54 @@ An example response can be seen below.
 ]
 ```
 
-## Mandatory Requirements
-- Must be completed in react (redux is optional)
-- Cannot use third party search select library
-- Behaviour and style should be the same as shown in the images below. You can choose which fields you want to use for the description. 
-- Items in the list should be selectable via a mouse click. On select, the character name should be written to the console.
-- A loader should be shown when the results are being queried from the API
+## Requirements
+
+- Must be completed in Vue or React (you're welcome to scaffold the project using a starter kit).
+- Cannot use a third party search select/autocomplete combobox library.
+- Behaviour and style roughly equal the [designs](#designs) below. You can choose which fields you want to use for the description. 
+- Items in the list should be selectable via a mouse click and Enter key. On select, the character name should be written to the console.
+- A spinner should be shown when the results are being queried from the API.
+
+## Bonus Points
+
+- Think about the accessibility requirements and considerations of building this solution.
+- Pay attention to the CSS architecture as much as the JavaScript architecture.
+
+## Guidance on Time Spent
+
+While you should be free to decide how much time you'd like to spend developing your solution, in the interests of not taking up all your time, we strongly advise to limit yourself to five hours or less.
 
 ## Submission
-The submission should be committed to Github and a link provided to the submission along with a readme on how to setup and run the solution. The solution is expected to incorporate good software development practices such as comments and tests.
+
+The submission should be committed to Github and a link provided to the submission along with a README on how to setup and run the solution. If you run out of time, detail in the README what else you would have liked to achieve.
```

Comment:
Did we agree we were going to create a repo for each candidate within the hireup org so they don't have to create a private repo in their own account and avoid having stuff publicly available? Should we detail the process here?